__tests__/app.test.js

```diff
@@ -127,11 +127,58 @@ describe("GET /api/articles", () => {
     return request(app)
       .get("/api/articles")
       .expect(200)
-      .then(({ body: { rows } }) => {
-        expect(rows).toBeSortedBy("created_at", {
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("created_at", {
           coearce: true,
           descending: true,
         });
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  it("Returns an array of comment objects for the chosen article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        comments.forEach((comment) => {
```

Comment:
It would also be beneficial to check that the correct number of comments have been returned here